lib/pages/wp-admin/wp-admin-sidebar.js

```diff
@@ -2,14 +2,16 @@
 
 import { By as by } from 'selenium-webdriver';
 
-import BaseContainer from '../../base-container';
 import * as driverManager from '../../driver-manager';
 import * as driverHelper from '../../driver-helper';
+import AsyncBaseContainer from '../../async-base-container';
 
-export default class WPAdminSidebar extends BaseContainer {
+export default class WPAdminSidebar extends AsyncBaseContainer {
 	constructor( driver ) {
-		driverHelper.refreshIfJNError( driver );
```

Comment:
Could you explain, why did you decide to move `refreshIfJNError` outside of page object?

Reply:
I was thinking the initialisation wouldn't happen in the right order but I think it'll be ok - I'll move it back

Reply:
Actually, yes I know - there can't be any async calls in the constructor - so I can't do await in there